queue_model_objects_v1.py

```diff
@@ -1129,7 +1137,7 @@ def __init__(self):
         self.wedge_prefix = str()
         self.run_number = int()
         self.suffix = str()
-        self.precision = str()
+        #self.precision = str()
```

Comment:
This should be fine. Static method is called from Session hwobj before __init__ and precision value is initialized. It is the same principle how the `base_directory, template, synchrotron_name`, etc are initialized.
If precison is added in `__init__` then it always is overwritten with empty str.